pandas/tools/tile.py

```diff
@@ -11,8 +11,9 @@
 import pandas.core.algorithms as algos
 import pandas.core.nanops as nanops
 from pandas.compat import zip
-
+from pandas import to_timedelta, to_datetime
 import numpy as np
+from pandas.types.common import is_datetime64_dtype, is_timedelta64_dtype
```

Comment:
can you put this one above the numpy import? (to keep the pandas imports grouped like it was before)